src/main/java/frc/trigon/robot/subsystems/sideshooter/SideShooter.java

```diff
@@ -0,0 +1,80 @@
+package frc.trigon.robot.subsystems.sideshooter;
+
+
+import com.ctre.phoenix6.hardware.TalonFX;
+import com.revrobotics.CANSparkMax;
+import edu.wpi.first.math.controller.ArmFeedforward;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.trajectory.TrapezoidProfile;
+import edu.wpi.first.math.util.Units;
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.FunctionalCommand;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+import java.lang.annotation.Target;
+
+public class SideShooter extends SubsystemBase {
+    private final static SideShooter INSTANCE = new SideShooter();
+    private final TalonFX shootingMotor = SideShooterConstants.SHOOTING_MOTOR;
+    private final CANSparkMax angleMotor = SideShooterConstants.ANGLE_MOTOR;
+
+    public static SideShooter getInstance() {
+        return INSTANCE;
+    }
+
+    private TrapezoidProfile angleMotorProfile = null;
+    private double lastAngleMotorProfileGeneration;
+
+    public Command getSetTargetShooterAngleCommand(Rotation2d targetAngle) {
+        return new FunctionalCommand(
+                () -> generateAngleMotorProfile(targetAngle),
+                this::setTargetAngleMotorProfileTime,
+                (interrupted) -> {},
+                () -> false,
+                this
+        );
+    }
```

Comment:
Extract this command into a `SideShooterCommands` class and create a command to set the target shooting voltage